web/src/elements/wizard/TypeCreateWizardPage.ts

```diff
@@ -0,0 +1,136 @@
+        return html`<div class="pf-l-grid pf-m-gutter">
+            ${this.types.map((type, idx) => {
+                const requiresEnterprise = type.requiresEnterprise && !this.hasEnterpriseLicense;
+                return html`<div
+                    class="pf-l-grid__item pf-m-3-col pf-c-card ${requiresEnterprise
+                        ? "pf-m-non-selectable-raised"
+                        : "pf-m-selectable-raised"} ${this.selectedType == type
+                        ? "pf-m-selected-raised"
+                        : ""}"
+                    tabindex=${idx}
+                    @click=${() => {
+                        if (requiresEnterprise) {
+                            return;
+                        }
+                        this.onSelect(type);
+                        this.selectedType = type;
+                    }}
+                >
+                    ${type.iconUrl
+                        ? html`<div class="pf-c-card__header">
+                              <div class="pf-c-card__header-main">
+                                  <img src=${type.iconUrl} alt=${msg(str`${type.name} Icon`)} />
+                              </div>
+                          </div>`
+                        : nothing}
+                    <div class="pf-c-card__title">${type.name}</div>
+                    <div class="pf-c-card__body">${type.description}</div>
+                    ${requiresEnterprise
+                        ? html`<div class="pf-c-card__footer">
+                              <ak-license-notice></ak-license-notice>
+                          </div> `
+                        : nothing}
+                </div>`;
+            })}
+        </div>`;
+    }
+
+    renderList(): TemplateResult {
+        return html`<form class="pf-c-form pf-m-horizontal">
+            ${this.types.map((type) => {
+                const requiresEnterprise = type.requiresEnterprise && !this.hasEnterpriseLicense;
+                return html`<div class="pf-c-radio">
+                    <input
+                        class="pf-c-radio__input"
+                        type="radio"
+                        name="type"
+                        id=${`${type.component}-${type.modelName}`}
+                        @change=${() => {
+                            this.onSelect(type);
+                        }}
+                        ?disabled=${requiresEnterprise}
+                    />
+                    <label class="pf-c-radio__label" for=${`${type.component}-${type.modelName}`}
+                        >${type.name}</label
+                    >
+                    <span class="pf-c-radio__description"
+                        >${type.description}
+                        ${requiresEnterprise
+                            ? html`<ak-license-notice></ak-license-notice>`
+                            : nothing}
+                    </span>
+                </div>`;
+            })}
+        </form>`;
+    }
+
+    render(): TemplateResult {
```

Comment:
`return (this.layout === TypeCreateWizardPageLayouts.grid) ? this.renderGrid() : this.renderList()` ? :-)

Reply:
Thought about that but I think this reads nicer